api/src/test/java/org/apache/iceberg/types/TestReadabilityChecks.java

```diff
@@ -134,7 +134,7 @@ private static Stream<Arguments> incompatibleTypesToVariant() {
                     Types.MapType.ofRequired(
                         1, 2, Types.StringType.get(), Types.IntegerType.get())),
                 Arguments.of(Types.ListType.ofRequired(1, Types.StringType.get()))),
-            Arrays.stream(PRIMITIVES).map(type -> Arguments.of(type)))
+            Arrays.stream(PRIMITIVES).map(Arguments::of))
```

Comment:
Please remove this change. It isn't related to this PR and can cause merge conflicts.